.travis.yml

```diff
@@ -134,15 +136,12 @@ install:
     set +e
 
 script:
-  - ./bootstrap
-  - ./configure $FFCONFIG
-  - make -j4
-  - make -C contrib/fonttools -j4
-  - make install
-  - if [ ! -z "$TRACE_CODE_COVERAGE" ]; then make check ; coveralls --gcov-options '\-bulp' --gcov $(which $GCOV) ; fi
-  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then make distcheck -j4; fi
-  - fontforge -version
-  - $PYTHON -c "import fontforge; import psMat; print(fontforge.__version__, fontforge.version()); fontforge.font();"
+  - if [ -n "$MODE" ] ; then ./bootstrap && ./configure $FFCONFIG ; fi
```

Comment:
Why is this conditional on $MODE being nonempty?

Reply:
`MODE` is unset when we want to skip the whole build, for example when in coverage mode but coveralls is not set up (see commit c1bfb98abdbb94c0b274f9e17617fdc7def108cc).